tests/openssl_supplier_test.cpp

```diff
@@ -41,15 +41,6 @@ TEST_F(OpenSSLSupplierTest, generate_key_RSA_3072) {
     ASSERT_TRUE(res);
 }
 
-TEST_F(OpenSSLSupplierTest, generate_key_RSA_7680) {
```

Comment:
Why was this removed?

Reply:
It took almost 1 minute for generation. Keys of that size are supposed to enter into usage somewhere after 2031, and it is a long way until there.